src/main/java/com/aerotrack/infrastructure/lambda/RefreshLambda/src/main/java/com/aerotrack/lambda/RefreshRequestHandler.java

```diff
@@ -1,156 +1,32 @@
-
-            URL url = URI.create(ryanairURL).toURL();
-            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
-            connection.setRequestMethod("GET");
-            connection.setRequestProperty("Cookie", RYANAIR_COOKIE);
-            connection.connect();
-
-
-            // Read the response
-            try(InputStream responseStream = connection.getInputStream()) {
-                String jsonResponse = IOUtils.toString(responseStream, StandardCharsets.UTF_8);
-
-                // Parse the response JSON and extract flight details
-                JSONObject flightDetails = new JSONObject(jsonResponse);
-                JSONArray trips = flightDetails.optJSONArray("trips");
-
-                if (trips != null && !trips.isEmpty()) {
-                    JSONObject trip = trips.getJSONObject(0);
-
-                    JSONArray dates = trip.optJSONArray("dates");
-                    if (dates != null && !dates.isEmpty()) {
-                        JSONObject date = dates.getJSONObject(0);
-
-                        JSONArray flights = date.optJSONArray("flights");
-                        if (flights != null && !flights.isEmpty()) {
-                            DynamoDbEnhancedClient enhancedClient = DynamoDbEnhancedClient.create();
-                            DynamoDbTable<Flight> customerMappedTable = enhancedClient.table(System.getenv("FLIGHT_TABLE"), TableSchema.fromBean(Flight.class));
-
-                            WriteBatch.Builder<Flight> builder = WriteBatch.builder(Flight.class).mappedTableResource(customerMappedTable);
-                            for (int i = 0; i < flights.length(); i++) {
-                                JSONObject flight = flights.getJSONObject(i);
-
-                                String timeFrom = flight.getJSONArray("timeUTC").getString(0);
-                                String timeTo = flight.getJSONArray("timeUTC").getString(1);
-                                double price = flight.getJSONObject("regularFare").getJSONArray("fares").getJSONObject(0).getDouble("amount");
-                                String flightNumber = flight.getString("flightNumber");
-
-                                Flight flightItem = new Flight(fromAirport + "-" + toAirport, timeFrom, timeTo, flightNumber, price);
-                                builder.addPutItem(flightItem);
-
-                            }
-
-                            BatchWriteItemEnhancedRequest batchWriteItemEnhancedRequest = BatchWriteItemEnhancedRequest.builder()
-                                    .writeBatches(builder.build())
-                                    .build();
-
-                            enhancedClient.batchWriteItem(batchWriteItemEnhancedRequest);
-                        }
-                    }
-                }
-            }
 
+        try {
+            new RefreshWorkflow(S3Client.create(), DynamoDbEnhancedClient.create()).handleRequest(logger);
```

Comment:
usare i create di AerotrackS3Client e del AerotrackDynamoDbClient in teoria ti risparmia tutte le dipendenze da aws di questa lambda (build più rapide)